src/app/helpers/spies.ts

```diff
@@ -65,7 +65,9 @@ export const propertiesServiceSpy = createSpyObj('PropertiesService', [
 ]);
 
 export const propertyDetailSpy = jasmine.createSpyObj('PropertyDetailService', [
-  'getProperty'
+  'getProperty',
+  'deletePropertyService',
+  'display_Modal'
```

Comment:
Hey @MBenedicte, I think this is not conventional js naming for methods, if thats the case, lets make it camelCase or kindly clarify why we are writing it this way. Thanks.

Reply:
worked on that @CryceTruly